include/deal.II/matrix_free/tensor_product_kernels.h

```diff
@@ -2328,6 +2332,479 @@ namespace internal
 
 
 
+  /**
+   * Internal evaluator for shape function in arbitrary dimension using the
+   * tensor product form of the basis functions.
+   *
+   * @tparam dim Space dimension in which this class is applied
+   * @tparam n_rows Number of rows in the transformation matrix, which corresponds
+   *                to the number of 1d shape functions in the usual tensor
+   *                contraction setting
+   * @tparam n_columns Number of columns in the transformation matrix, which
+   *                   corresponds to the number of 1d shape functions in the
+   *                   usual tensor contraction setting
+   * @tparam Number Abstract number type for input and output arrays
+   * @tparam Number2 Abstract number type for coefficient arrays (defaults to
+   *                 same type as the input/output arrays); must implement
+   *                 operator* with Number and produce Number as an output to
+   *                 be a valid type
+   */
+  template <int dim,
+            int n_rows,
+            int n_columns,
+            typename Number,
+            typename Number2>
+  struct EvaluatorTensorProduct<evaluate_raviart_thomas,
+                                dim,
+                                n_rows,
+                                n_columns,
+                                Number,
+                                Number2>
+  {
+    static constexpr unsigned int n_rows_of_product =
+      Utilities::pow(n_rows, dim); // WRONG!
```

Comment:
Add comment why wrong.

Reply:
If necessary, we could write a `static constexpr` with the correct content, `Utilities::pow(n_rows, dim - 1) * (n_rows + 1)`.

Reply:
Don't think these are used anywhere, so maybe I should just remove them?